src/components/m-table-edit-cell.js

```diff
@@ -3,8 +3,8 @@ import * as React from "react";
 import PropTypes from "prop-types";
 import TableCell from "@material-ui/core/TableCell";
 import CircularProgress from "@material-ui/core/CircularProgress";
-import { fade } from "@material-ui/core/styles/colorManipulator";
-import withTheme from "@material-ui/core/styles/withTheme";
+import { alpha } from "@material-ui/core/styles/colorManipulator";
```

Comment:
These deep access are actually [forbidden](https://next.material-ui.com/guides/minimizing-bundle-size/#option-1), Can we shorten them?

Reply:
I added a new eslint rule that should catch this moving forward.

https://github.com/material-table-core/core/blob/efb68b7a17d14f3ed482de457c01c01f055be658/.eslintrc#L4-L9